src/Editor/Image/Panel.tsx

```diff
@@ -0,0 +1,186 @@
+import React, { FC, useState, useMemo } from 'react';
+import { CropperRef, Cropper } from 'react-advanced-cropper';
+import { EditorArgs } from '../EditorArgs';
+import { EditorOptions } from '../EditorOptions';
+import { useWhppt } from '../../Context';
+import { WhpptGalleryTag, WhpptInput } from '../../ui/components';
+import { Gallery } from '../../Gallery';
+import { aspectRatios } from './AspectRatios';
+import { getLandscapeRatio, getPortraitRatio } from './helpers';
+import { DevicePicker } from './DevicePicker';
+import { ImageDataSize, PageImageData, ImageData } from './../../Gallery/Model/Image';
+
+type Orientation = 'landscape' | 'portrait';
+
+export type ImageEditorOptions = EditorOptions & { sizes?: string[] };
+
+export const WhpptImageEditor: FC<EditorArgs<PageImageData, ImageEditorOptions>> = ({ value, onChange, options }) => {
+  const { toggleSettingsPanel } = useWhppt();
+
+  const [device, setDevice] = useState<string>('desktop');
+  const [orientation, setOrientation] = useState<Orientation>('landscape');
+
+  const selectedDevice = useMemo(() => value && value[device], [device, value]);
```

Comment:
Damn I think I missed all the times it was used in the file. It would have been better.
Im always happy for you to correct me on these. 
I guess either way works. so no need to make any changes.